pkgs/development/libraries/mkldnn/default.nix

```diff
@@ -0,0 +1,37 @@
+{ stdenv, fetchFromGitHub, cmake, substituteAll, bash, autoPatchelfHook }:
+
+stdenv.mkDerivation rec {
+  pname = "mkldnn";
```

Comment:
It looks like as of release 1.0, Intel is renaming the library to `dnnl`: https://github.com/intel/mkl-dnn#deep-neural-network-library-dnnl

Perhaps we should update the `pname` and top-level attrname to match?

Reply:
I doubt people are aware of this -- pytorch build flags are also hardcoded to MKLDNN. I would expect the right move is to do as you suggest and also include a top-level alias to mkldnn so that this is backwards compatible.